eng/pipelines/templates/stages/1es-redirect.yml

```diff
@@ -22,21 +22,29 @@ extends:
   ${{ else }}:
     template: v1/1ES.Unofficial.PipelineTemplate.yml@1ESPipelineTemplates
   parameters:
-    settings:
-      skipBuildTagsForGitHubPullRequests: true
-    sdl:
-      ${{ if and(parameters.GenerateBaselines, eq(variables['Build.SourceBranchName'], 'main'), eq(variables['System.TeamProject'], 'internal')) }}:
-        autobaseline:
-          isMainPipeline: true
-          disableAutoBaselineOnNonDefaultBranches: true
-          enableForGitHub: true
-      sourceAnalysisPool:
-        name: azsdk-pool
-        image: windows-2022
-        os: windows
-      psscriptanalyzer:
-        compiled: true
-        break: true
-      policy: M365
-
-    stages: ${{ parameters.stages }}
\ No newline at end of file
+    ${{ if and(parameters.Use1ESOfficial, eq(variables['System.TeamProject'], 'internal')) }}:
+      settings:
+        skipBuildTagsForGitHubPullRequests: true
+      sdl:
+        ${{ if and(parameters.GenerateBaselines, eq(variables['Build.SourceBranchName'], 'main'), eq(variables['System.TeamProject'], 'internal')) }}:
+          autobaseline:
+            isMainPipeline: true
+            disableAutoBaselineOnNonDefaultBranches: true
+            enableForGitHub: true
+        sourceAnalysisPool:
+          name: azsdk-pool
+          image: windows-2022
+          os: windows
+        psscriptanalyzer:
+          compiled: true
+          break: true
+        policy: M365
+      stages: ${{ parameters.stages }}
+    ${{ else }}:
+      sdl:
+        enableAllTools: false
+        sourceAnalysisPool:
```

Comment:
Let's add a comment here linking to the issue about removing this entire stage.

Reply:
Added comment.